doc/02-Installation.md

```diff
@@ -104,17 +98,41 @@ apk update
 
 #### RHEL/CentOS Notes <a id="package-repositories-rhel-notes"></a>
 
-The packages for RHEL/CentOS depend on other packages which are distributed
-as part of the [EPEL repository](https://fedoraproject.org/wiki/EPEL).
-
+Our packages are build on and with packages from the **[EPEL repository](https://fedoraproject.org/wiki/EPEL)**.
+Please enable it prior installing the Icinga packages.
+ 
 CentOS 7/6:
-```
-yum install epel-release
-```
 
-If you are using RHEL you need enable the `optional` repository and then install
-the [EPEL rpm package](https://fedoraproject.org/wiki/EPEL#How_can_I_use_these_extra_packages.3F).
+    yum install epel-release
```

Comment:
Please keep the format with the three backticks in the docs.

Reply:
Is there a reason for that?

Reply:
The entire documentation uses that format, and we should keep it that way.